interop/client/client.go

```diff
@@ -79,6 +79,7 @@ var (
 	soakMinTimeMsBetweenRPCs               = flag.Int("soak_min_time_ms_between_rpcs", 0, "The minimum time in milliseconds between consecutive RPCs in a soak test (rpc_soak or channel_soak), useful for limiting QPS")
 	soakRequestSize                        = flag.Int("soak_request_size", 271828, "The request size in a soak RPC. The default value is set based on the interop large unary test case.")
 	soakResponseSize                       = flag.Int("soak_response_size", 314159, "The response size in a soak RPC. The default value is set based on the interop large unary test case.")
+	soakNumThreads                         = flag.Int("soak_num_threads", 1, "The number of threads for concurrent execution of the soak tests (rpc_soak or channel_soak). The default value is set based on the interop large unary test case.")
```

Comment:
Note that Go doesn't really concern itself with threads, so this is wrong in the context of Go.  You probably can't change this because it has cross-language implications, but this should have been thought out more and reviewed by more folks before implementation started.

Reply:
You’re absolutely right about the thread issue and the cross-language implications. The Java version was merged first, which influenced the approach. I appreciate your suggestion and will make sure to keep it in mind going forward. Thank you for your insight!